src/Pulse.php

```diff
@@ -520,11 +520,17 @@ public function css(string|Htmlable|array|null $css = null): string|self
      */
     public function js(): string
     {
-        if (($content = file_get_contents(__DIR__.'/../dist/pulse.js')) === false) {
+        if (
+            ($livewire = @file_get_contents(__DIR__.'/../../../livewire/livewire/dist/livewire.js')) === false &&
+            ($livewire = @file_get_contents(__DIR__.'/../vendor/livewire/livewire/dist/livewire.js')) === false) {
+            throw new RuntimeException('Unable to load the Livewire JavaScript.');
+        }
```

Comment:
These two are required to support applications with Pulse installed and local testing.

Reply:
Would it be better to use `base_path()`?

Reply:
I thought that as well, however TestBench changes the `base_path` to its own directory. This fix is specifically for those scenarios so it doesn't work here unfortunately.